### PR TITLE
[4.0] Apply formatting to token.xml

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Apply coding style.
Change radio buttons to switcher.

### Testing Instructions
Code review.

or 

Edit an admin account.
Go to Joomla Token tab.
See radio buttons display as switcher.

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 431a38745616ce8de4c23de65e181e183802f5c3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28482">issues.joomla.org/tracker/joomla-cms/28482</a>.</sub>

File: plugins/user/token/token/token.xml [hunk 1]
```diff
@@ -1,57 +1,70 @@
...
+				class="input-xxlarge"
+			/>
+			<field
+				name="enabled"
+				type="radio"
+				label="PLG_USER_TOKEN_ENABLED_LABEL"
+				description="PLG_USER_TOKEN_ENABLED_DESC"
+				layout="joomla.form.field.radio.switcher"
+				default="1"
+				filter="integer"
```
Comment (wilsonge) [inline plugins/user/token/token/token.xml:51]: If you're adding filters here can you please change the PHP to strict equality checks?

Comment (Quy) [reply plugins/user/token/token/token.xml:51]: Changed for `reset`, but cannot find this one. Please advise file to change or I can revert the `filter` attribute.

**Comment by brianteeman** [comment]
@quy really sorry I didnt see this PR

**Comment by Quy** [comment]
No worries. It is all good. Now I can be one of the testers. 👍

**Pull request closed without merging**